Missing header install

```
/space/pvelesko/zero-rk/applications/cfd_plugin/interfaces/hipsolver_rf_manager/hipsolver_rf_manager.h:6:10: fatal error: 'hipsolver/hipsolver.h' file not found
    6 | #include <hipsolver/hipsolver.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~
1 error generated when compiling for .
```

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -65,6 +65,10 @@ install(TARGETS hipsolver
 )
 install(FILES
     ${CMAKE_CURRENT_BINARY_DIR}/include/hipsolver.h
+    DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/hipsolver
+)
+install(FILES
+    ${CMAKE_CURRENT_BINARY_DIR}/include/hipsolver.h
     DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/
 )
 

